concordium-consensus/src/Concordium/Skov/Update.hs

```diff
@@ -373,7 +373,7 @@ addBlock block = do
                         check "Baker key claimed in block did not match actual baker key" (_bakerSignatureVerifyKey == blockBakerKey block) $ do
                             -- Update the seed state with the block nonce
                             let newSeedState = updateSeedState (blockSlot block) (blockNonce block) parentSeedState
-                            let ts = blockTransactions block
+                            ts <- getTransactionVerificationResults $ blockTransactions block
```

Comment:
It seems less than ideal to have to look up the verification result here again.

Would it not be possible to retain this information from storeBlock and propagate it here?